### PR TITLE
Portfolio Mobile Menu Project

### PR DESCRIPTION
**Pull request by Carshy**
In this project, I implemented mobile menu on my portfolio project using my recently acquired JavaScript knowledge. During the project, I recreated the hamburger and used JavaScript to make it interactive when the user clicks. For instance, when the user clicks the hamburger, the navigation menu appears with the nav links to navigate to different sections of the portfolio. This project was helpful and I have added the following expertise on my skillset:
* Learned how to use basic JavaScript syntax to manipulate DOM elements
* Use JavaScript events
* Design the hamburger using CSS
* Using HTML id to navigate to portfolio sections

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -699,8 +687,79 @@ form .contacts-lists {
...
+    z-index: 10;
+    width: 375px;
+    height: 100%;
+    padding: 100px 0 0 28px;
+  }
+
+  #menu-x {
+    margin-left: 91%;
+    font-size: 16px !important;
+  }
```
Comment (RolandM99) [inline styles/style.css]: - [ ] Kindly, make sure that the margins of your close-button fit exactly the margins on the Figma design. 

|Your Design|Figma Design|
|-------------|---------------|
|image|image|

File: styles/style.css [hunk 2]
```diff
@@ -699,8 +687,79 @@ form .contacts-lists {
...
+  a {
+    text-decoration: none;
+  }
+
+  .nav-menu {
+    position: fixed;
+    gap: 0;
+    display: flex;
+    flex-direction: column;
+    width: 375px;
```
Comment (RolandM99) [inline styles/style.css]: - [ ] Great job by implementing the `mobile-menu`. the functionality work as expected but something doesn't yet fit the requirement. Once, we click on the close button, the navigation menu disappears but once we scroll down the page that menu still appears but in a small format. To go on the next project, kindly fix this issue.
you can see it on the screenshot below:
image

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi  @Carshy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Carshy** [comment]
Hello @RolandM99. Thank for reviewing my code I fixed the requirements as requested. After clicking the x icon, the mobile menu disappears but while scrolling down it reappears and I have tried to fix it since yesterday with many of my peers and the challenge keeps reappearing. I am looking forward to fixing the issue and I am researching massively to find where the js is not responding since I am a beginner.

File: index.html [hunk 3]
```diff
@@ -25,17 +25,12 @@ <h1><a href="#">My Logo</a></h1>
...
                 <div class="nav-items">
-                    <ul>
-                        <li><a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li><a href="#">Contact</a></li>
+                    <ul class="nav-menu">
+                        <li><a class="nav-link" id="menu-x" href="#">X</a></li>
+                        <li><a class="nav-link" href="#portfolio">Portfolio</a></li>
+                        <li><a class="nav-link" href="#about">About</a></li>
+                        <li><a class="nav-link" href="#contact">Contact</a></li>
```
Comment (keroloussamy) [inline index.html:32]: - [ ] Kindly as asked by the previous reviewer, your menu still appears when I scroll down as you see on the pic, To fix it you can just hide it, and when you click on the menu button make it appear, and when click on the close button make it disappear again.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Carshy, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

 ## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Carshy** [comment]
Hello, thanks for taking your time to review my project. @keroloussamy I appreciate your inputs and I managed to fix the changes as requested. I learned a lot thanks.

**Review by ishimwezachee — APPROVED** [review]
Hi @Carshy  and @Nedramrez , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**